libraries/appnexusKeywords/anKeywords.js

```diff
@@ -1,5 +1,17 @@
-import {_each, getValueString, isArray, isStr, mergeDeep, isNumber} from '../../src/utils.js';
+import {_each, deepAccess, getValueString, isArray, isStr, mergeDeep, isNumber} from '../../src/utils.js';
 import {getAllOrtbKeywords} from '../keywords/keywords.js';
+import {CLIENT_SECTIONS} from '../../src/fpd/oneClient.js';
+
+const ORTB_SEGTAX_KEY_MAP = {
+  0: null,
```

Comment:
this is unnecessary (assuming `0` is never a valid segtax) - you'd get `undefined` instead of `null` but your check would still work.

Reply:
Will push an update